pytools/__init__.py

```diff
@@ -2907,6 +2912,34 @@ def sphere_sample_fibonacci(
 # }}}
 
 
+# {{{ strtobool
+
+def strtobool(val: str, default: Optional[bool] = None) -> bool:
```

Comment:
I could see returning default if `val` is `None`, but returning it if the value is not understood is unintuitive and potentially dangerous IMO.

Reply:
Done in 83d121e